x-pack/plugins/security_solution/common/api/detection_engine/model/rule_schema/rule_schemas.schema.yaml

```diff
@@ -74,6 +74,11 @@ components:
         throttle:
           $ref: './common_attributes.schema.yaml#/components/schemas/RuleActionThrottle'
 
+        response_actions:
```

Comment:
Previously we discussed that only specific rule types support response actions. Adding it to base `BaseOptionalFields` schema will make it possible passing `response_actions` via rule CRUD API. Does it mean all rule types support response actions now?

Reply:
Thanks Maxim 👍 
Yes, you got it right. And it's my fault that I brought the confusion. Previously we were more than sure that not all rules support agent.id in alerts, but apparently it is depending fully on the source of alert. We discussed this more in slack and I tested with more rules and apparently we can enable that in ALL rule types. 

The `shouldShowResponseActions` is going to be there just for the first release of serverless (intermediate release) to have the logic on backend, but not expose response_actions in the UI of newly added types of rules. But after a week with another serverless release I am going to remove this check totally. 

How does it sound?